### ISSUE TITLE
A11Y error due to tabindex > 0

### ISSUE THREAD

**Issue by Willibaur** [knownasilya/ember-toggle#118]
Thanks for this awesome addon.

Below you will see an `A11Y` error related, it was catched by `ember-A11Y-testing` it seems an easy one to fix on :point_right: [x-toggle/component.js](https://github.com/knownasilya/ember-toggle/blob/d4468f2c6c1ac126edc0030a80a81d1d519c57b8/addon/components/x-toggle/component.js#L12)

![image](https://user-images.githubusercontent.com/12820932/46530759-b6a75480-c892-11e8-926f-1d0ad2112b3d.png)

I'm happy to create a PR if you agree this should be `tabindex: '0'`.

Thanks

**Comment by knownasilya** [comment]
Could probably just remove the tabindex all together and users can apply those as needed.

**Comment by Willibaur** [comment]
I'm not sure, as a general A11Y rule, it should be `tabindex: 0`, so what about to set it to that value by default within the addon, but give the option to the user to modify as required @knownasilya like we do with `offLabel` property ?

**Comment by knownasilya** [comment]
The user can already set it, e.g. `{{x-toggle tabindex='0'}}`

**Comment by knownasilya** [comment]
I'll change it to '0'

**Comment by Willibaur** [comment]
Thanks for the fix @knownasilya

**Comment by knownasilya** [comment]
Released in 5.3.1
